docs/source/usage.rst

```diff
@@ -76,44 +81,55 @@ For information on installing ``repo2docker``, see :ref:`install`.
    Docker must be running on your machine in order to build images
    with ``repo2docker``.
 
-Building an image
-~~~~~~~~~~~~~~~~~
+The simplest invocation of ``repo2docker`` performs two steps:
+
+1. builds a Docker image from a git repo
+2. runs a Jupyter server within the image
 
-The simplest invocation of ``repo2docker`` builds a Docker image
-from a git repo, then runs a Jupyter server within the image
-so you can explore the repository's contents.
-You can do this with the following command::
+This two step process enable you to build an image and run it so you can
+explore the repository's contents.
+
+The **command** used is::
+
+  juptyer-repo2docker <URL to repo>
+
+where ``<URL to repo>`` provides a URL to the source repository.
```

Comment:
do we want to cram in a comment that a local path also works or put that somewhere else? There is a lot here already, but then this is our only usage guide page :-/

Reply:
good point, why not just make it `<URL-or-path to repo>``